WishList/WishList/Friends/SearchFriendsScreen.swift

```diff
@@ -23,12 +23,12 @@ struct SearchFriendsScreen: View {
                     .multilineTextAlignment(.center)
                 
             }else {
-                List(fullFakeFriends) { friend in
+                List(friends) { friend in
```

Comment:
Since `friends` should become `people`, change the `friend` for `person`, for the same reason.